ipycanvas/canvas.py

```diff
@@ -575,6 +574,9 @@ class Canvas(_CanvasBase):
     #: This property has no effect on Safari, see https://bugs.webkit.org/show_bug.cgi?id=198416
     filter = Unicode("none")
 
+    #: (bool) Indicates if scaled images are smoothed. Default to True
+    image_smoothing_enabled = Enum([True, False], default_value=True)
```

Comment:
This can be defined as bool:

```suggestion
    image_smoothing_enabled = Bool(True)
```